include/silo/common/nucleotide_symbols.h

```diff
@@ -0,0 +1,93 @@
+#ifndef SILO_H
+#define SILO_H
+
+#include <array>
+#include <iostream>
+
+#include <spdlog/spdlog.h>
+
+namespace silo {
+
+static constexpr unsigned GENOME_LENGTH = 29903;
+
+// https://www.bioinformatics.org/sms/iupac.html
+enum class NUCLEOTIDE_SYMBOL {
+   GAP,  // -, GAP
+   A,    // Adenine
+   C,    // Cytosine
+   G,    // Guanine
+   T,    // (or U) Thymine (or Uracil)
+   R,    // A or G
+   Y,    // C or T
+   S,    // G or C
+   W,    // A or T
+   K,    // G or T
+   M,    // A or C
+   B,    // C or G or T
+   D,    // A or G or T
+   H,    // A or C or T
+   V,    // A or C or G
+   N,    // any base
+   UNDEFINED = -1
```

Comment:
Why is this still around? Can't we also use an optional where it is used?

Reply:
Should be deleted.